src/commands/write-swagger.ts

```diff
@@ -0,0 +1,20 @@
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { writeFileSync } from 'fs';
+
+import { AppModule } from '~/app.module';
+
+const writeSwaggerSpec = async (): Promise<void> => {
+  const app = await NestFactory.create(AppModule, { logger: false });
```

Comment:
@prashantasdeveloper wether this is a test module, or a normal one, this still wants to call out when we configure the module. We had the issue of this getting masked from local `.env` files

I don't want to call out to a non local node in CI. The easiest fix would be to start a docker container, but that seems overkill when its not actually needed.

Our mocks are dependent on having jest. Its tempting, but seems like tech debt to exploit DI with a mock though.

I am no sure on a clean way to solve this and don't want to get more side tracked from CDD at this point. No urgency on this PR, so I can revisit later this week. I am certainly appreciating the simplicity of the write + commit approach, but that has draw backs when other people are running this imo.